cmd/loop/loopin.go

```diff
@@ -76,6 +76,18 @@ func loopIn(ctx *cli.Context) error {
 		return err
 	}
 
+	// For loop in, the fee estimation is handed to lnd which tries to
+	// construct a real transaction to sample realistic fees to pay to the
+	// HTLC. If the wallet doesn't have enough funds to create this TX, we
+	// don't want to fail the quote. But the user should still be informed
+	// why the fee shows as 0.
+	if quote.Status == looprpc.QuoteResponse_MINER_FEE_ESTIMATION_FAILED {
+		fmt.Printf("Warning: Miner fee estimation not " +
+			"possible, lnd has insufficient funds to " +
+			"create a sample transaction for selected " +
+			"amount.\n")
+	}
```

Comment:
Does it make sense to proceed with the swap without funds?

Reply:
No, not really. Returning error now instead.